### ISSUE TITLE
[Bug] [ConsoleSinkWriter] java.lang.ClassCastException: [I cannot be cast to [Ljava.lang.Object

### ISSUE THREAD

**Issue by zhaomin1423** [apache/seatunnel#2570]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

java.lang.ClassCastException: [I cannot be cast to [Ljava.lang.Object

### SeaTunnel Version

2.1.3-SNAPSHOT

### SeaTunnel Config

```conf
none
```

### Running Command

```shell
run SeaTunnelApiExample
```

### Error Exception

```log
java.lang.ClassCastException: [I cannot be cast to [Ljava.lang.Object;
	at org.apache.seatunnel.connectors.seatunnel.console.sink.ConsoleSinkWriter.fieldToString(ConsoleSinkWriter.java:63)
	at org.apache.seatunnel.connectors.seatunnel.console.sink.ConsoleSinkWriter.write(ConsoleSinkWriter.java:49)
	at org.apache.seatunnel.connectors.seatunnel.console.sink.ConsoleSinkWriter.write(ConsoleSinkWriter.java:31)
	at org.apache.seatunnel.translation.spark.sink.SparkDataWriter.write(SparkDataWriter.java:58)
	at org.apache.seatunnel.translation.spark.sink.SparkDataWriter.write(SparkDataWriter.java:37)
	at org.apache.spark.sql.execution.datasources.v2.DataWritingSparkTask$$anonfun$run$3.apply(WriteToDataSourceV2Exec.scala:118)
	at org.apache.spark.sql.execution.datasources.v2.DataWritingSparkTask$$anonfun$run$3.apply(WriteToDataSourceV2Exec.scala:116)
	at org.apache.spark.util.Utils$.tryWithSafeFinallyAndFailureCallbacks(Utils.scala:1394)
	at org.apache.spark.sql.execution.datasources.v2.DataWritingSparkTask$.run(WriteToDataSourceV2Exec.scala:146)
	at org.apache.spark.sql.execution.datasources.v2.WriteToDataSourceV2Exec$$anonfun$doExecute$2.apply(WriteToDataSourceV2Exec.scala:67)
	at org.apache.spark.sql.execution.datasources.v2.WriteToDataSourceV2Exec$$anonfun$doExecute$2.apply(WriteToDataSourceV2Exec.scala:66)
	at org.apache.spark.scheduler.ResultTask.runTask(ResultTask.scala:90)
```

### Screenshots

<img width="1289" alt="image" src="https://user-images.githubusercontent.com/49054376/187503391-819516aa-0c4c-4a50-a8cb-cda0c1a858ee.png">

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by laglangyue** [comment]
Please sync your fork，this bug has be fixed in PR #2545,
and the PR has approval 10 hours ago.

**Comment by Hisoka-X** [comment]
Closed by #2545
